alzueta.yaml

```diff
@@ -0,0 +1,1849 @@
+      -2.40527335e+04, 1.90175132]
+    note: |-
+      FAB/JAN05
+      H298 =-44.33 kcal/mol [FAB/JAN05]
+      S298 = 60.07 cal/mol/K [FAB/JAN05]
+      Cp [FAB/JAN05] (polyfit RAS/GLA08a)
+  transport:
+    model: gas
+    geometry: nonlinear
+    well-depth: 498.0
+    diameter: 3.59
+    rotational-relaxation: 2.0
+    note: WKM =OCHO LLNL
+- name: CH2O
+  composition: {C: 1, H: 2, O: 1}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 6000.0]
+    data:
+    - [4.77187231, -9.76266155e-03, 3.70121918e-05, -3.76921617e-08, 1.31326646e-11,
+      -1.43798106e+04, 0.696585679]
+    - [2.91332516, 6.70039926e-03, -2.55520684e-06, 4.27795142e-10, -2.44072521e-14,
+      -1.44622358e+04, 7.43822875]
+    note: |-
+      ATcT3E
+       H2CO <g> ATcT ver. 1.122, DHf298 = -109.188 ?? 0.099 kJ/mol - fit JAN17
+  transport:
+    model: gas
+    geometry: nonlinear
+    well-depth: 498.0
+    diameter: 3.59
+    rotational-relaxation: 2.0
+- name: HCO
+  composition: {C: 1, H: 1, O: 1}
+  thermo:
+    model: NASA7
+    temperature-ranges: [200.0, 1000.0, 6000.0]
+    data:
+    - [3.97074749, -1.49121608e-03, 9.54041776e-06, -8.82720349e-09, 2.67645129e-12,
+      3842.03291, 4.44660361]
+    - [3.85781113, 2.6411395e-03, -7.44177294e-07, 1.2331323e-10, -8.88958718e-15,
+      3616.42883, 3.92451197]
+    note: |-
+      ATcT3E
+      REF=ATcT C 2011}  Max Lst Sq Error Cp @ 6000 K 0.49%.
+       from Xiong et al., Combust. Flame 161 (2014) 885???897
+      3-HCOH                  C   1H   2O   1    0G   298.150  6000.000 1000.00      1
+       0.62184153E+01 0.18701090E-02-0.17117529E-06-0.44242689E-10 0.63999362E-14    2
+       0.23486822E+05-0.97996712E+01 0.36513441E+01 0.46834047E-02 0.11223500E-05    3
+      -0.80289814E-09-0.77469460E-12 0.24561953E+05 0.49211311E+01                   4
+      1-HCOH                  C   1H   2O   1    0G   298.150  6000.000 1000.00      1
+       0.57648468E+01 0.21295433E-02-0.19747641E-06-0.51074096E-10 0.74327861E-14    2
+       0.10621689E+05-0.77241540E+01 0.32130404E+01 0.32250737E-02 0.24648079E-05    3
+       0.15449875E-08-0.27946393E-11 0.11899702E+05 0.76449280E+01                   4
+      CHOH trans Hydroxymethylene trans  SIGMA=1  STATWT=1  IA=0.2970  IB=2.3197
+      IC=2.6167  Nu=3634,2827,1536,1330,1224,1099  REF=Burcat G3B3  HF298=108.16+/-.43
+      kJ  REF=Ruscic ATcT D 2013  {HF298=106.47+/-8 kJ  REF=Burcat G3B3}  Max Lst Sq
+      Error Cp @ 6000 K 0.45%.
+      CHOH trans  Hydr  T12/14C  1.H  2.O  1.   0.G   200.000  6000.000  B  30.02598 1
+      3.63500546E+00 5.45479239E-03-1.91285077E-06 3.03882849E-10-1.79904440E-14    2
+      5.28968001E+04 4.34543184E+00 4.72862471E+00-1.02082828E-02 4.21639528E-05    3
+      -4.64774522E-08 1.72559970E-11 5.31829869E+04 1.69093263E+00 5.44279146E+04    4
+       HCO <g> ATcT ver. 1.122, DHf298 = 41.803 ?? 0.099 kJ/mol - fit JAN17
+  transport:
```

Comment:
It would be nice to clean up some of these "note" fields that are mostly just commented out thermo entries. In this case, just the last line is actually a reference for the HCO data that should be preserved.